LondonBritishLibrary/orient/BLorient2083.xml

```diff
@@ -0,0 +1,2470 @@
+                                                    <locus from="216va"/>
+                                                    <title ref="LIT2715John">John 20:20-23</title>
+                                                </msItem>
+                                                <msItem xml:id="ms_i1.14.2.12">
+                                                    <locus from="217vb"/>
+                                                    <title ref="LIT1544Gebrah#Directives">Liturgical direction and prayers</title>
+                                                    <explicit xml:lang="gez">ተፈጸመ፡ በዝየ፡ በጸጋሁ፡ ለክርስቶስ፡ መጽሐፈ፡ ኅብረተ፡ ቃላት፡ ዘይደሉ፡ ለአንብቦ፡ በሰሙነ፡ ሕማማት፡ 
+                                                        እምቅዳሚት፡ ለስንበተ፡ ሆሣዕና፡ ሰርክ፡ እስከ፡ እሁድ፡ ትንሣኤ፡ ሰርክ።</explicit>
+                                                </msItem>
+                                                <colophon xml:lang="gez" xml:id="coloph1">
+                                                    <locus target="#218ra"/>
+                                                    <note>Colophon containing the names of the scribe
+                                                        <persName role="scribe" ref="PRS14424TasfaIyasus">Tasfā ʾIyasus</persName> and of
+                                                        <persName role="patron" ref="PRS14423Malakotawit">Malakotāwit</persName>, who commissioned 
+                                                        the manuscript.</note>
+                                                </colophon>
+                                            </msItem>
+                            </msItem>
+                      </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">218</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>350</height>
+                                        <width>307</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="22"/>
+                            </layoutDesc>
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Neat gʷǝlḥ script, decreasing in height from 6 mm at the beginning to about 4 mm at the end of the manuscript.</desc>
+                                <date notBefore="1800" notAfter="1899" resp="PRS8999Strelcyn"/>
```

Comment:
Is the date here a typo? The origin date suggested for the manuscript is notBefore="1694" notAfter="1706", and gwelh script would usually also point to an earlier time.